virttest/utils_libvirt/libvirt_secret.py

```diff
@@ -3,12 +3,69 @@
 """
 
 import logging
+import re
+
+from aexpect import remote
+
+from avocado.core import exceptions
 
 from virttest import virsh
+from virttest.utils_test import libvirt
+from virttest.libvirt_xml import secret_xml
 
 LOG = logging.getLogger('avocado.' + __name__)
 
 
+def create_secret(sec_dict=None, remote_args=None):
+    """
+    Create a secret with 'virsh secret-define'
+
+    :param sec_dict: dict to create secret
+    :param remote_args: Parameters for remote host
+    :return: UUID of the secret
+
+    Example:
+
+        set dict in cfg file:
+            secret_dict = {'secret_ephemeral': 'no',
+                            'secret_private': 'yes',
+                            'description': 'secret desc',
+                            'usage': 'volume',
+                            'volume': '/path/to/volume'}
+
+        secret_dict = eval(self.params.get("secret_dict", '{}'))
```

Comment:
Are L29 and L36 necessary? can we delete them?

Reply:
done